content/en/docs/contribute/localization.md

```diff
@@ -214,27 +277,30 @@ To find source files for your target version:
 2. Select a branch for your target version from the following table:
     Target version | Branch
     -----|-----
-    Next version | [`dev-{{< skew nextMinorVersion >}}`](https://github.com/kubernetes/website/tree/dev-{{< skew nextMinorVersion >}})
     Latest version | [`master`](https://github.com/kubernetes/website/tree/master)
-    Previous version | `release-*.**`
+    Previous version | [`release-{{< skew prevMinorVersion >}}`](https://github.com/kubernetes/website/tree/release-{{< skew prevMinorVersion >}})
+    Next version | [`dev-{{< skew nextMinorVersion >}}`](https://github.com/kubernetes/website/tree/dev-{{< skew nextMinorVersion >}})
 
-The `master` branch holds content for the current release `{{< latest-version >}}`. The release team will create `{{< release-branch >}}` branch shortly before the next release: v{{< skew nextMinorVersion >}}.
+The `master` branch holds content for the current release `{{< latest-version >}}`. The release team will create a `{{< release-branch >}}` branch before the next release: v{{< skew nextMinorVersion >}}.
```

Comment:
nit: The release team creates a `{{< release-branch >}}` branch before the next release: v{{< skew nextMinorVersion >}}.

Reply:
Let's fix this in a separate PR. The wording here is close to the previous version.